lib/round.rb

```diff
@@ -0,0 +1,46 @@
+class Round 
```

Comment:
It's good practice to put an empty line in between each new method definition. It makes it a little easier to read.